app/controllers/agentAppointment/AppointAgentController.scala

```diff
@@ -36,19 +32,17 @@ import play.api.Logger
 import play.api.data.Forms.{number, _}
 import play.api.data.{Form, FormError}
 import play.api.i18n.MessagesApi
-import play.api.libs.json.Json
 import play.api.mvc.{Action, AnyContent, Request, Result}
 import repositories.SessionRepo
 import services.AppointRevokeAgentService
-import uk.gov.hmrc.http.HeaderCarrier
+
 import uk.gov.voa.play.form.ConditionalMappings.mandatoryIfEqual
 
 import scala.concurrent.{ExecutionContext, Future}
 
 class AppointAgentController @Inject()(
                                         representations: PropertyRepresentationConnector,
                                         accounts: GroupAccounts,
-                                        propertyLinks: PropertyLinkConnector,
                                         agentsConnector: AgentsConnector,
                                         authenticated: AuthenticatedAction,
                                         appointRevokeService: AppointRevokeAgentService,
```

Comment:
The service should ideally be named after the business case it is filling, Aka Agent Relationship, I missed in last PR.

Reply:
done